test/WebSites/InlineConstraintsWebSite/Constraints/IsbnDigitScheme10Constraint.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.AspNet.Http;
+using Microsoft.AspNet.Routing;
+
+namespace InlineConstraintsWebSite.Constraints
+{
+    public class IsbnDigitScheme10Constraint : IRouteConstraint
+    {
+        private readonly bool _turnOn;
+
+        public IsbnDigitScheme10Constraint(bool turnOn)
```

Comment:
Hmm then maybe I'd make it a more realistic example, such as an option that allows optional hyphens in the number or something. So with `allowDashes=true` it would allow `123-456-789-X` and with `allowDashes=false` it would require `123456789X`.